### ISSUE TITLE
White Screen

### ISSUE THREAD

**Issue by jonahromero** [azarrias/sprite-sheet-splitter#2]
Running it on windows with LOVE. Opened the image dialogue (working), and seems to be running the game loop, however, it isn't drawing anything. Instead it's just a blank white screen.

**Comment by azarrias** [comment]
Hi there, when opening the application, the Open file window opens. Meanwhile, the main window will remain blank until you choose your desired sprite sheet. Once that you open it, the spritesheet should draw on the main window.

**Comment by jonahromero** [comment]
Hello, the sprite sheet should be a png correct? I selected the image and clicked open in the dialogue. I even tried commenting out the source code, and just used the default sprite sheet that was provided. Are there any other reasons the screen would remain white?

**Comment by azarrias** [comment]
Yes, the file format must be supported by love (png or tga). That part of the code is all here: https://github.com/azarrias/sprite-sheet-splitter/blob/master/src/SlicerGUI.lua#L269 The only reason I can think of for the image file not opening is a wrong version of LOVE interpreter or problems with access rights to the file path. I don't have this development environment setup at the moment, have you tried running the release version?

**Comment by jonahromero** [comment]
Hey my apologies for the last response. I looked up the version and it seems to be "LOVE 11.4 (Mysterious Mysteries)." Is that the same one you're using? I have file permissions/have tried it on different files. Is the imgui dll you linked maybe outdated? I'll keep looking to see if I can fix it in the meantime.
